Week3/server.js

```diff
@@ -0,0 +1,49 @@
+"use strict";
+
+const bodyParser = require("body-parser");
+const express = require("express");
+const db = require("./queries.js");
+const app = express();
+
+app.use(bodyParser.json());
+
+app.post("/create-user/:id", (req, res) => {
+  db.createUser(req.params.id, req.body.user_name, req.body.user_last_name);
```

Comment:
Try also to validate your inputs and to try and catch possible error from the ```createUser()``` method call. 

For example, what would happen if I made a POST request to '/create-user/4' but without sending body or without user_name in the body? =)